### PR TITLE
fix: use private rpc endpoints with paraspell

### PR DESCRIPTION
**Pull request by noahjoeris**
This PR shifts from using public nodes to private nodes when making paraspell calls. 
I changed it for balance, fees and tx calls.

### CODE REVIEW

File: app/src/hooks/useFees.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { getNativeToken } from '@/registry'
 import useNotification from '@/hooks/useNotification'
 import { Chain } from '@/models/chain'
 import { NotificationSeverity } from '@/models/notification'
 import { getCoingekoId, Token } from '@/models/token'
 import { Fees } from '@/models/transfer'
+import { getNativeToken } from '@/registry'
```
Comment (NunoAlexandre) [inline app/src/hooks/useFees.tsx:6]: Do we need this import here?

**Review by NunoAlexandre — APPROVED** [review]
Easy and effective 🙌🏻 there's a merge conflict, let me know if help is needed

File: app/src/hooks/useFees.tsx (hunk 1 above)
Comment (noahjoeris) [reply app/src/hooks/useFees.tsx]: It is gone after the merge conflict resolve

Comment (noahjoeris) [reply app/src/hooks/useFees.tsx]: nope build failed without it. Will add it again.

**Review by NunoAlexandre — APPROVED** [review]
LG!!

**Pull request merged**